packages/@react-spectrum/label/chromatic/HelpText.chromatic.tsx

```diff
@@ -9,68 +9,99 @@
-  )
-  .add(
-    'disabled',
-    () => render({description: 'Password must be at least 8 characters.', isDisabled: true})
-  )
-  .add(
-    'labelAlign: end',
-    () => render({
-      description: 'Password must be at least 8 characters.',
-      labelAlign: 'end'
-    })
-  )
-  .add(
-    'labelPosition: side',
-    () => render({
-      description: 'Password must be at least 8 characters.',
-      labelPosition: 'side'
-    })
-  )
-  .add(
-    'no visible label',
-    () => render({
-      label: null,
-      'aria-label': 'Password',
-      description: 'Password must be at least 8 characters.'
-    })
-  )
-  .add(
-    'custom width',
-    () => render({
-      label: 'Password',
-      description: 'Password must be at least 8 characters.',
-      width: '100px'
-    })
-  )
-  .add(
-    'custom width, labelPosition: side',
-    () => render({
-      label: 'Password',
-      description: 'Password must be at least 8 characters.',
-      width: '440px',
-      labelPosition: 'side'
-    })
-  );
+type HelpTextStory = ComponentStoryObj<typeof TextField>;
+
+export default {
+  title: 'HelpText',
+  component: TextField,
+  args: {
+    label: 'Password',
+    description: 'Password must be at least 8 characters.'
+  }
+} as ComponentMeta<typeof TextField>;
+
+export let Default: HelpTextStory = {
+  name: 'description'
+};
+
+export const ErrorMessage = {
+  ...Default,
+  args: {
+    errorMessage: 'Create a password with at least 8 characters.',
+    validationState: 'invalid'
+  },
+  name: 'error message'
+};
+
+export const Disabled = {
+  ...Default,
+  args: {
+    isDisabled: true
+  },
+  name: 'disabled'
+};
+
+export const LabelAlignEnd = {
+  ...Default,
+  args: {
+    labelAlign: 'end'
+  },
+  name: 'labelAlign: end'
+};
+
+export const LabelPositionSide = {
+  ...Default,
+  args: {
+    labelPosition: 'side'
+  },
+  name: 'labelPosition: side'
+};
+
+export const NoVisibleLabel = {
+  ...Default,
+  args: {
+    label: null,
+    'aria-label': 'Password'
+  },
+  name: 'no visible label'
+};
+
+export const CustomWidth = {
+  ...Default,
+  args: {
+    width: '100px'
+  },
+  name: 'custom width'
+};
+
+export const CustomWidthLabelPositionSide = {
+  ...Default,
+  args: {
+    width: '440px',
+    labelPosition: 'side'
+  },
+  name: 'custom width, labelPosition: side'
+};
 
-function render(props: SpectrumTextFieldProps = {}) {
-  return (
-    <TextField label="Password" {...props} />
-  );
-}
+export const ContainerWithTextAlignmentSet = {
+  render: () => (
+    <Flex
+      direction="column"
+      gap="size-200"
+      UNSAFE_style={{
+        textAlign: 'center'
+      }}>
+      <TextField label="Password" description="Enter a single digit number." />
+      <TextField
+        label="Password 2"
+        errorMessage="Create a password with at least 8 characters."
+        validationState="invalid" />
+    </Flex>
+  ),
+  name: 'container with text alignment set'
+};
```

Comment:
Let's break this into two and use decorators so that we don't need a custom renderer and we can just use args (I probably have some typos in this, but this is the general idea)

```suggestion
export const ContainerWithTextAlignmentSetDescription = {
  args: {
    label: 'Password'
    description: 'Enter a single digit number'
  },
  name: 'textAlign center description',
  decorator: [textAlignDecorator]
};
export const ContainerWithTextAlignmentSetError = {
  args: {
    label: 'Password'
    errorMessage: 'Create a password with at least 8 characters.'
  },
  name: 'textAlign center errorMessage',
  decorator: [textAlignDecorator]
};

function textAlignDecorator(Story) {
   return (
      <Flex
      direction="column"
      gap="size-200"
      UNSAFE_style={{
        textAlign: 'center'
      }}>
        <Story />
      </Flex>
    )
}
```

Reply:
I assumed we wanted two TextFields next to each other.